manifests/gcp_marketplace/guide.md

```diff
@@ -0,0 +1,65 @@
+# Deploy Kubeflow Pipelines from Google Cloud Marketplace
+
+You can go to [Google Cloud Marketplace](https://console.cloud.google.com/marketplace) to deploy Kubeflow Pipelines via UI.
+You can find the instances in [Pipelines Console](http://pantheon.corp.google.com/ai-platform/pipelines).
+
+## Cluster
+You can choose an existing cluster or create a new cluster. Creating a new cluster from Marketplace config page doesn't allow
+you to customize the cluster setting. If you want to customize the cluster,
+please go to [Google Kubernetes Engine](https://console.cloud.google.com/kubernetes/list) to create one.
+
+Please notice that you only can `install one Kubeflow Pipelines into one cluster`.
+
+## Namespace
+Please specify a [Kubenetes namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/).
+
+## App instance name
+Please specify a name for this instance. It's possible you install muliple instances to muliple clusters.
+Please specify a friendly name so that you can well manage it in [Pipelines Console](http://pantheon.corp.google.com/ai-platform/pipelines).
+
+## GCP Service Account credentials
+It's used to call other GCP services like Google Cloud Storage, Dataflow etc.
+
+This deployment requires a service account to use for authentication when calling other GCP services. 
+Specify the base64-encoded credentials for the service account you want to use. 
+You can get these credentials by running the following command in a terminal window. 
+
+```
+$ gcloud iam service-accounts keys create application_default_credentials.json --iam-account [your-service-account] && cat application_default_credentials.json | base64
+```
```

Comment:
This command will create a new key under the service account. Please note that a single service account can only have 10 keys.